client/tests/Banner.test.js

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import Banner from '../src/Banner';
 import { shallow } from 'enzyme';
+import sinon from 'sinon';
 
 describe('Unit test for <Banner /> Component', () => {
   it('should render a banner with class name "banner"', () => {
```

Comment:
One small thing, you may have labelled this test incorrectly. It's description says it should be testing a class name but it looks like it is texting the text of the button.